web/server/README.md

```diff
@@ -96,6 +96,24 @@ openssl req -newkey rsa:2048 -nodes -sha512 -x509 -days 365 -keyout key.pem -out
 openssl speed rsa2048 rsa4096
 ```
 
+### Select TLS version
+
+Since version 1.21 Netdata accepts `tls key` and `tls certificate` as replacement of the options `ssl key` and `ssl certificate`, 
+ but you do not need to change your `netdata.conf` case you are using old `ssl` variables, because old options are kept due compatibility.
+
+Netdata also brings two new options related to TLS, now you can specify the TLS version and the ciphers that you want to use:
```

Comment:
```suggestion
You can also specify the TLS version and the ciphers that you want to use:
```

Reply:
I will mark this and the next as resolved, for the motive I already given.